test_reporting/sai_swss_invocations.py

```diff
@@ -0,0 +1,338 @@
+import json
+import os
+import pathlib
+from pathlib import Path
+
+from datetime import datetime
+
+from os import listdir, name
+from os.path import isfile, join
+import shutil
+
+from report_data_storage import KustoConnector
+
+'''
+for more detail, you can see
+https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/373699/sairedis-log-scanner
+'''
+
+
+ngsdevice_type = "ToRRouter"
+sai_path = "/data/sonic-mgmt-int/SAI/inc/"
```

Comment:
This folder is self defined or cloned automatically?
 Need to check the existence and report error message with details like where is the repo and command, or clone it in your code